cmd/juju/crossmodel/list.go

```diff
@@ -34,20 +34,15 @@ options:
        - charm
 
 Examples:
-    $ juju list-offers
-    $ juju list-offers vendor:
-    $ juju list-offers --url vendor:/u/ibm
-    $ juju list-offers --interface db2
+    $ juju offers
+    $ juju offers user/model
+    $ juju offers --url user/model
```

Comment:
No, I meant `juju offers user/model` vs `juju offers --url user/model`. Oh - are they equivalent, with `--url` is the default (and `--interface` being the other way to query once it's implemented)?